### DISCUSSION TITLE
Which is the font used in README screenshot?

### DISCUSSION THREAD

**Discussion by shreyas-a-s** [skbolton/embark#64] [General]
It looks neat and simple. I would like to know which one it is.

**Comment by skbolton** [comment]
I believe its Lilex. I switch fonts a lot though so I could be wrong.

**Comment by shreyas-a-s** [comment]
Not sure if it's [Lilex](https://github.com/mishamyrt/Lilex) but I am loving Lilex btw. Thanks for sharing the name.
